src/libraries/Microsoft.Extensions.Logging.Abstractions/gen/LoggerMessageGenerator.Emitter.cs

```diff
@@ -152,7 +152,7 @@ private void GenStruct(LoggerMethod lm, string nestedIndentation)
 ");
                 GenVariableAssignments(lm, nestedIndentation);
                 _builder.Append($@"
-                {nestedIndentation}return $""{lm.Message}"";
+                {nestedIndentation}return $""{ConvertEndOfLineAndQuotationCharactersToEscapeForm(lm.Message)}"";
```

Comment:
refer to: 
https://github.com/dotnet/runtime/blob/c545cf8bfbffd85ce0fd132374d258c3ea7f790d/src/libraries/Microsoft.Extensions.Logging.Abstractions/gen/LoggerMessageGenerator.Emitter.cs#L541